### PR TITLE
oopsy: Sephirot EX

### PR DESCRIPTION
**Pull request by JLGarber**
Largely self-explanatory. I could use criticism on the Force Against mistake text, since even after I cut it down to what it is now, it's still really long.

I'm not sure what we want to do with the non-tank + no-debuff tower trigger, since #4530 is still floating around. I would be okay removing it, but it's going to be moderately important once this is released for Unreal.

### CODE REVIEW

File: ui/oopsyraidsy/data/03-hw/trial/sephirot-ex.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+      id: 'SephirotEX Spirit',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '157B', source: 'Sephirot' }),
+      condition: (data, matches) => data?.force?.[matches.target] === '3ED',
+      mistake: (_data, matches) => {
+        return {
+          type: 'warn',
+          blame: matches.target,
+          text: {
+            en: 'Took green; orange debuff',
```
Comment (quisquous) [inline ui/oopsyraidsy/data/03-hw/trial/sephirot-ex.ts:82]: What about just calling all of these custom warnings just the ability name? I think the fact that it gets listed as a mistake means "you shouldn't have done this" and matches what oopsy does elsewhere.

File: ui/oopsyraidsy/data/03-hw/trial/sephirot-ex.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+          },
+        };
+      },
+    },
+    {
+      // Only tanks should take towers without a Force debuff.
+      id: 'SephirotEX Fiendish Wail Non-Tank',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '1576', source: 'Sephirot' }),
+      condition: (data, matches) => !data.party.isTank(matches.target) && data?.force?.[matches.target] === undefined,
```
Comment (quisquous) [inline ui/oopsyraidsy/data/03-hw/trial/sephirot-ex.ts:122]: I think for now just leave this as-is and assume that we'll (I'll?) fix this sometime soon.

Also, how should this handle BLU?

**Review by quisquous — COMMENTED** [review]
Sorry for the extremely belated review here.  Thank you for taking a look at this!

File: ui/oopsyraidsy/data/03-hw/trial/sephirot-ex.ts (hunk 1 above)
Comment (JLGarber) [reply ui/oopsyraidsy/data/03-hw/trial/sephirot-ex.ts:82]: Sure, that makes sense. My concern was that the ability names are kinda ambiguous (Life Force/Spirit), but I guess that's true (for non-Hebrew speakers) of basically all the Kabbalistic terms in the encounter anyway.

File: ui/oopsyraidsy/data/03-hw/trial/sephirot-ex.ts (hunk 2 above)
Comment (JLGarber) [reply ui/oopsyraidsy/data/03-hw/trial/sephirot-ex.ts]: Fair point, we can just not call Blue Mages if they take towers without a debuff. Fortunately that's the only role-based trigger we'll need for this encounter!

**Pull request merged**